### DISCUSSION TITLE
Grid: FixedHeader="True" prevents the THeadCssClass parameter from being applied

### DISCUSSION THREAD

**Discussion by MaMon1963** [vikramlearning/blazorbootstrap#631] [Q&A] [answered]
By setting the FixedHeader parameter to TRUE to keep the grid header fixed on the page, the set colors are lost and the header appears with black text on a white background. By setting the FixedHeader parameter to FALSE the colors return correct.
These are my settings:
```
<Grid @ref="grid"
       TItem="DataDTO"
       Class="table table-hover table-bordered table-striped align-middle"
       DataProvider="DataProvider"
       FixedHeader="false"
       AllowFiltering="true"
       AllowPaging="true"
       AllowSorting="true"
       AllowSelection="false"
       PageSize="@pageSize"
       FiltersTranslationProvider="GridFiltersTranslationProvider"
       THeadCssClass="bg-success text-white"
       HeaderRowCssClass="bg-primary text-white"
       EmptyText="Non ci sono dati da mostrare"
       PageSizeSelectorVisible="true"
       PageSizeSelectorItems="@(new int[] { 9,20,50 })"
       PaginationItemsTextFormat="{0} - {1} di {2} elementi"
       ItemsPerPageText="Elementi per pagina"
       Responsive="true">
```

**Comment by gvreddy04** [comment, marked as answer]
@MaMon1963 Thank you for trying BlazorBootstrap. There is a known issue. I have provided an interim solution for this. We will fix this in the upcoming release. Please go through the issue https://github.com/vikramlearning/blazorbootstrap/issues/621.

**Reply by MaMon1963** [reply]
I want to thank you for your availability.  Your solution actually solves the background color problem but for header and filter apply the same background colors.

**Reply by gvreddy04** [reply]
@MaMon1963 Please share a sample GitHub repository to reproduce your issue with minimal steps. I'll check what's missing.

**Reply by MaMon1963** [reply]
Hi Vikram, I was wondering if, with the release of version 3, there was any possibility of overcoming the problem of the fixed table header coloring that you somehow solved temporarily, with your suggestions but with the promise of a permanent solution. 
What can you tell us about it?
